src/app/ngrx/reducers/selectors/downloads-chart.selectors.spec.ts

```diff
@@ -42,6 +42,8 @@ describe('Downloads Chart Selectors', () => {
         metricsPropertyName, metrics: ep1Downloads}));
       store.dispatch(new ACTIONS.CastlePodcastMetricsSuccessAction({
         id: podcast.id, metricsPropertyName, metrics: podDownloads}));
+      store.dispatch(new ACTIONS.ChartTogglePodcastAction({
```

Comment:
This test was failing for me since the podcast was not charted, so I added this toggle action. It could be that this is a smell indicating an incompatibility with the non-entity implementation.

Reply:
I see what's going on here... It's related to the change to the `charted` toggle. The first time the entity is saved to state, it doesn't get the `charted` property because of the includes logic. When the application has a LOAD then a  SUCCESS action, `charted: true` is added. For the tests, we're just cutting right to the SUCCESS action, so it's not getting `charted`.